plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx

```diff
@@ -39,7 +39,10 @@ function splitFormData(url: string | undefined) {
     if (url) {
       const parsed = new URL(`https://${url}`);
       host = parsed.host;
-      owner = parsed.searchParams.get('owner') || undefined;
+      owner =
+        parsed.searchParams.get('owner') || allowedOwners
```

Comment:
Do we really want this behaviour to default to the first of the `allowedOwners`?

Reply:
I did it cause this is what is done with allowedHost, you think it should be different ?